src/util/cursor.js

```diff
@@ -30,43 +30,43 @@ function isValidCursorConfig (config) {
  * Example: (createdDate) isn't enough because two rows can have the same value
  * then (createdDate, id) is a good candidate for cursor
  * @param {Object}   obj - cursor will be created based on this object
- * @param {Object[]} config
- * @param {Object}   config[i]
- * @param {String}   config[i].prop - object property to encode
- * @param {String}   config[i].type - allowed value: 'number', 'boolean', 'date', 'string'
+ * @param {Object[]} props
+ * @param {Object}   props[i]
+ * @param {String}   props[i].name - prop name to encode
+ * @param {String}   props[i].type - allowed value: 'number', 'boolean', 'date', 'string'
  */
-function createCursor (obj, config) {
-  if (!config) throw new Error('Missing cursor config')
-  if (!isValidCursorConfig(config)) throw new Error('Invalid cursor config')
+function createCursor (obj, props) {
+  if (!props) throw new Error('Missing cursor props')
```

Comment:
Maybe full word `properties` is better in error messages intended for users.

Reply:
Actually this error is intended to be displayed to us as API designers, because it warns us that the `cursorProps` passed by [`performListQuery`](https://github.com/stelace/stelace/blob/3880051b4d30aea2bcbf9146c61636544428e20f/src/util/listQueryBuilder.js#L295-L298) or [`performHistoryQuery`](https://github.com/stelace/stelace/blob/3880051b4d30aea2bcbf9146c61636544428e20f/src/util/listQueryBuilder.js#L645-L647) has an incorrect format.

This error will never be displayed to API consumers. It will be revealed thanks to failing cursor pagination test.

Reply:
Does this also applies to other occurences?

Reply:
Yes it does.